packages/manager/modules/pci/src/projects/project/storages/object-storage/users/add/user-add/user-add.html

```diff
@@ -0,0 +1,68 @@
+<form novalidate name="user-add-form">
+    <oui-modal
+        data-heading="{{ 'pci_projects_project_users_add_title' | translate }}"
+        data-primary-label="{{:: 'pci_projects_project_users_add_submit_label' | translate }}"
+        data-primary-action="$ctrl.submit($ctrl.userModel,$ctrl.model)"
+        data-secondary-action="$ctrl.cancel()"
+        data-secondary-label="{{:: 'pci_projects_project_users_add_cancel_label' | translate }}"
+        data-on-dismiss="$ctrl.goBack()"
+        data-primary-disabled="$ctrl.disable"
+        data-secondary-disabled="$ctrl.disable"
+        data-loading="$ctrl.isLoading"
+    >
+        <oui-radio
+            model="$ctrl.model"
+            name="addExistingUser"
+            on-change="$ctrl.onChange($ctrl.addExistingUser)"
+            value="$ctrl.addExistingUser"
+        >
+            <span
+                data-translate="pci_projects_project_users_add_existing_user"
+            ></span>
+        </oui-radio>
+        <oui-field
+            data-size="xl"
+            data-ng-if="$ctrl.model === $ctrl.addExistingUser"
+        >
+            <oui-select
+                data-name="userList"
+                data-model="$ctrl.userModel"
+                data-items="$ctrl.allUserList"
+                data-match="username"
+            >
+                <span style="float:left" data-ng-bind="$item.username"></span>
+                <small>
+                    <span
+                        style="float:right"
```

Comment:
**nitpick:** Inline styles should not be used.

Reply:
Taken care Anoop